nette/notejam/app/presenters/templates/Note/delete.latte

```diff
@@ -0,0 +1,9 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	<p>Are you sure you want to delete the note?</p>
+	{form deleteNoteForm}
```

Comment:
Imho it's nicer to use `<form n:name="deleteNoteForm">` since the editors understand HTML it better than "obscure" macros (which every framework has different)